### PR TITLE
chore: set minimal kubernetes version to 1.19

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Set minimal kubernetes version to 1.19

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/20182

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by MichalMaler — APPROVED** [review]

**Review by rkratky — APPROVED** [review]

**Comment by MichalMaler** [comment]
@tolusha dont merge this yet. Need to do the backport for 7.34.x. Will merge myself when ready :)

**Comment by tolusha** [comment]
It is not needed to be backported

**Pull request merged**
